setup.py

```diff
@@ -0,0 +1,29 @@
+from distutils.core import setup, Extension
+from os import path
+from glob import glob
+
+setup(
+    name = 'windows-container',
+    ext_modules = [
+        Extension('winc',
+            define_macros = [
+                # see https://msdn.microsoft.com/en-us/library/windows/desktop/ms683219(v=vs.85).aspx
+                ('PSAPI_VERSION', '1'),
+                # FIXME: _HAS_EXCEPTIONS only has effect when /MT
+                # But python builds it with /MD
+                # ('_HAS_EXCEPTIONS', '0'),
+                # For future?
+                ('BINDING_PYTHON_EXPORTS', None),
+                # Use UNICODE
+                ('_UNICODE', None),
```

Comment:
I think this is not needed. The code uses the XxxW form to call API, bypassing the unicode macro.